src/infrastructure/transport/notes-api/index.ts

```diff
@@ -0,0 +1,40 @@
+import Transport from '../index';
+import ApiResponse from '../types/ApiResponse';
```

Comment:
move this type to the `/transport/notes-api/types/`

Reply:
done